app/views/admin/settings/discovery/show.html.haml

```diff
@@ -42,6 +42,11 @@
   .fields-group
     = f.input :peers_api_enabled, as: :boolean, wrapper: :with_label, recommended: :recommended
 
+  %h4= t('admin.settings.security.federation_authentication')
+
+  .fields-group
+    = f.input :authorized_fetch, as: :boolean, wrapper: :with_label, label: t('admin.settings.security.authorized_fetch'), warning_hint: authorized_fetch_overridden? ? t('admin.settings.security.authorized_fetch_disabled_hint') : nil, hint: t('admin.settings.security.authorized_fetch_hint'), disabled: authorized_fetch_overridden?
```

Comment:
`admin.settings.security.authorized_fetch_disabled_hint` would be more explicit as `admin.settings.security.authorized_fetch_overridden_hint`